### ISSUE TITLE
[dev.icinga.com #10225] Host notification type is PROBLEM but should be RECOVERY

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga2#3439]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/10225

**Created by mwaldmueller on 2015-09-28 07:46:44 +00:00**

Assignee: _mfriedrich_
Status: _Resolved_  (closed on _2016-03-11 08:35:05 +00:00_)
Target Version: _2.4.4_
Last Update: _2016-05-03 13:17:56 +00:00 (in Redmine)_

    Icinga Version: 2.3.10
    Backport?: Already backported
    Include in Changelog: 1

---

When a host or service recovers from a warning/critical state the notification.type should be RECOVERY. Instead of the notification type has the value PROBLEM.

Here's an output of a default email notification with the subject: "PROBLEM - testhost is UP"

    ***** Icinga  *****

    Notification Type: PROBLEM

    Host: testhost
    Address: 127.0.0.1
    State: UP

    Date/Time: 2015-09-25 14:52:35 +0200

    Additional Info: PING WARNING - DUPLICATES FOUND! Packet loss = 0%, RTA = 0.33 ms

    Comment: [] 

**Changesets**

_2016-03-10 13:32:57 +00:00_ by _mfriedrich_ c0e175d641022a71e8b65a9f65af5bb3920018e0

    Fix host recovery notifications for warning states
    
    refs #10225

_2016-03-11 08:29:07 +00:00_ by _mfriedrich_ 5b6a6f86b157a4f4e6da74e6d464575ed666ddd0

    Fix host recovery notifications for warning states
    
    fixes  #10225

_2016-03-11 14:58:13 +00:00_ by _mfriedrich_ ee7af887c8d4814236cab4ea2eff89cd2c8011a1

    Fix host recovery notifications for warning states
    
    fixes  #10225

_2016-03-15 08:47:59 +00:00_ by _mfriedrich_ 3bd68487632c475839f6de79684608a9bf85f285

    Refactor patch for host recovery notifications
    
    refs #10225

_2016-03-15 12:11:01 +00:00_ by _mfriedrich_ f386920e60ed0c9878b2676971f2f14a0fb971d0

    Refactor patch for host recovery notifications
    
    refs #10225

---

**Relations**:

* relates [#10225](https://dev.icinga.com/issues/10225)
* relates [#11339](https://dev.icinga.com/issues/11339)
